test/integration/unpublishing_test.rb

```diff
@@ -67,9 +68,13 @@ class UnpublishingTest < ActiveSupport::TestCase
 
     unpublish(@published_edition, unpublishing_params)
 
-    assert_publishing_api_put_content(@published_edition.unpublishing.content_id, { format: 'unpublishing' }, 2)
-    assert_publishing_api_publish(@published_edition.unpublishing.content_id, { "locale" => "en", "update_type" => "major" })
-    assert_publishing_api_publish(@published_edition.unpublishing.content_id, { "locale" => "fr", "update_type" => "major" })
+    assert_publishing_api_put_content(@published_edition.unpublishing.content_id,
+                                      request_json_includes({ format: 'unpublishing' }),
+                                      2)
+    assert_publishing_api_publish(@published_edition.unpublishing.content_id,
+                                  { "locale" => "en", "update_type" => "major" })
```

Comment:
Do these keys and the ones below need to be strings and thus hash rocket style key/value?